src/components/feedback/Dialog.tsx

```diff
@@ -196,7 +196,7 @@ const Dialog = function Dialog({
 
   return (
     <Wrapper
-      visible={transitionComponentVisible}
+      direction={transitionComponentVisible ? 'in' : 'out'}
```

Comment:
I'm curious: this `Wrapper` may be a reference to a `TransitionComponent` functional component or it might be a `Fragment`. Do `Fragment`s happily take any props without grumbling? Presumably typechecking would bark if not...I just haven't thought about it before.